public/components/test.js

```diff
@@ -1,5 +1,5 @@
 import './game/details/details_test';
-//import './navigation/navigation_test';
+// import './navigation/navigation_test';
```

Comment:
I wasn't sure why it was left in master so I felt weird about removing, but it can be removed.